docs/excel/custom-functions-overview.md

```diff
@@ -327,6 +327,11 @@ function refreshTemperature(thermometerID){
 }
 ```
 
+## Co-Authoring
+Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature is supported by custom functions. If your workbook uses a custom function, your sharing partner will be prompted to install the custom function's add-in in order to display the results. Once both people sharing the document have installed the add-in, all custom function behavior should work normally.
```

Comment:
```suggestion
Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature works with custom functions. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through co-authoring.
```

Reply:
Just a few consistency recommendations like using "colleague" similar to support docs like here https://support.office.com/en-us/article/collaborate-on-excel-workbooks-at-the-same-time-with-co-authoring-7152aa8b-b791-414c-a3bb-3024e46fb104